loops.js

```diff
@@ -51,3 +67,21 @@
 // >> 9
 // ==========================================
 
+for (i=0;i<10; i++){
+    if (i < 3)
+        console.log(i);
+    else if (i <= 5){
+        console.log(">",i);
+    }
+    else if (i <= 9){
+        console.log(">>", i);
+    }
+
+    // Ook mogelijk
+    // for (let j = 0; j < 10; j++) {
+    //     j < 3 ? console.log(j) : j < 6 ? console.log('<' , j) : console.log('<<', j)
+    // }
+
+
```

Comment:
nu ik jouw code zie snap ik je feedback op mijn code bij deze opdracht. Wat handig dat niet meer gecheckt hoeft te worden of het groter of gelijk aan 3 is. Ik had gedacht dat een tweede if de eerste zou overschrijven als er enkel kleiner of gelijk aan 5 zou staan. 

tussen ; i < 10; mogen spaties en er mag staan let i = 0 ipv enkel i=0

Je 'ook mogelijk' is denk ik nog niet af.

Reply:
De ook mogelijk is wel af, alleen lijkt dat zo. dit is de "verkorte" manier van het opschrijven. maar wilde de andere niet verwijderen zodat als ik het ooit zou vergeten deze 2 zou kunnen terugvinden haha!